### PR TITLE
Updated and adjusted language files

### PR DESCRIPTION
**Pull request by Somebodyisnobody**
* Description adjusted (IDS_LANG_INFO)
* Statements made more precised (IDS_MSG_CMD_HOSTONLY and IDS_MSG_NOPLAYERSELECTED)
* Removed IDS_MSG_NOUNREGEDIT, IDS_MENU_REGISTRATION, IDS_MSG_EDITORREGONLY, IDS_PRC_NOSWSCENARIO, IDS_SEC_UNREGUSER and IDS_TYPE_KEYFILE (were forgotten in 005b6f25f448f651c551e93f19cd9a0a98a367e1)
* Removed double IDS_CTL_NONE, IDS_TEXT_VIEW and IDS_TEXT_DESKTOP in US-file (inserted in 66b40452c375ef43118d9eb23822672a63cc84d2)

### CODE REVIEW

File: planet/System.c4g/LanguageDE.txt [hunk 1]
```diff
@@ -875,14 +873,13 @@ IDS_MSG_NOMATS=Keine Materialdefinitionen verf
 IDS_MSG_NOMODIFYFAIRCREW=Faire Crew kann nicht ge�ndert werden: Durch Szenario vorgegeben!
 IDS_MSG_NOMODIFYORIGINAL=%s kann nicht aktualisiert werden, da es Teil eines Originalpakets ist.
 IDS_MSG_NOPARTICIPATE_DESC=%s von der n�chsten Runde ausnehmen.
-IDS_MSG_NOPLAYERSELECTED=Es sind keine Spieler ausgew�hlt. Bitte zuerst in der Spielerauswahl die Teilnehmer f�r diese Runde aktivieren.
+IDS_MSG_NOPLAYERSELECTED=Es sind keine Spieler ausgew�hlt. Bitte zuerst in der Spielerauswahl im Hauptmen� die Teilnehmer f�r diese Runde aktivieren.
```
Comment (Fulgen301) [inline planet/System.c4g/LanguageDE.txt]: Please don't change the encoding.

File: planet/System.c4g/LanguageDE.txt [hunk 2]
```diff
@@ -783,7 +782,7 @@ IDS_MSG_CHATDISCLAIMER=Chat - Nutzungsbedingungen
 IDS_MSG_CLEARTHEMESSAGEBOARD=Alle Nachrichten l�schen.
 IDS_MSG_CLIENT=Client
 IDS_MSG_CMD_ABORT_NOCOUNTDOWN=Countdown l�uft nicht!
-IDS_MSG_CMD_HOSTONLY=Kein Host? Versagt!
+IDS_MSG_CMD_HOSTONLY=Nur der Host kann das machen.
```
Comment (Fulgen301) [inline planet/System.c4g/LanguageDE.txt]: "kann das machen" sounds a bit too childish for me, maybe "kann diesen Befehl ausführen"? (Why ever this message needs to be changed)

File: planet/System.c4g/LanguageDE.txt (hunk 1 above)
Comment (Somebodyisnobody) [reply planet/System.c4g/LanguageDE.txt]: Resolved in IRC chat: GitHub bug, compare 
PR-Commit https://raw.githubusercontent.com/legacyclonk/LegacyClonk/afbbba9ed30db4e2d6bf9e634361a0237c4cab28/planet/System.c4g/LanguageDE.txt

Master https://raw.githubusercontent.com/legacyclonk/LegacyClonk/master/planet/System.c4g/LanguageDE.txt

File: planet/System.c4g/LanguageDE.txt (hunk 2 above)
Comment (Somebodyisnobody) [reply planet/System.c4g/LanguageDE.txt]: Changed 70b108f944a8692a99adb0246c551e55affb6dc5

**Pull request merged**